timeline-chart/src/time-graph-container.ts

```diff
@@ -75,9 +75,10 @@ export class TimeGraphContainer {
     }
 
     // if canvas size has changed displayWidth need to be updated for zoomfactor
-    reInitCanvasSize(newWidth: number){
-        this.application.renderer.resize(newWidth, this.config.height);
+    reInitCanvasSize(newHeight: number, newWidth: number){
```

Comment:
For backward-compatibility, you should put newHeight as second parameter

Reply:
Also, for backward-compatibility, make the second argument optional with an ok default value

Reply:
Also add a space before {